x-pack/plugins/triggers_actions_ui/public/application/components/builtin_action_types/servicenow/servicenow_params.tsx

```diff
@@ -48,16 +48,11 @@ const ServiceNowParamsFields: React.FunctionComponent<
 > = ({ actionConnector, actionParams, editAction, index, errors, messageVariables }) => {
   const { incident, comments } = useMemo(
     () =>
-      actionParams.subActionParams ?? {
-        incident: {
-          short_description: null,
-          description: null,
-          severity: null,
-          urgency: null,
-          impact: null,
-        },
+      actionParams.subActionParams ??
```

Comment:
@YulNaumenko this fixes the bug with validation on `short_description`

Reply:
Short description validation is resolved! 👍